### PR TITLE
[DOCS] Fixes deprecation message for Geo-polygon query

### PR DESCRIPTION
**Pull request by lockewritesdocs**
The deprecation message for Geo-polygon query was not formatted correctly, causing it to display incorrectly. This PR fixes the formatting error.
image

Preview link: https://elasticsearch_71141.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/query-dsl-geo-polygon-query.html

### CODE REVIEW

File: docs/reference/query-dsl/geo-polygon-query.asciidoc [hunk 1]
```diff
@@ -4,8 +4,7 @@
 <titleabbrev>Geo-polygon</titleabbrev>
 ++++
 
-deprecated[7.12.0,"Use <<query-dsl-geo-shape-query>> instead,
-where polygons are defined in geojson or wkt."]
+deprecated:[7.12.0] Use <<query-dsl-geo-shape-query>> instead, where polygons are defined in GeoJSON or http://docs.opengeospatial.org/is/18-010r7/18-010r7.html[Well-Known Text (WKT)].
```
Comment (jrodewig) [inline docs/reference/query-dsl/geo-polygon-query.asciidoc:7]: Drive-by nit:

I think this should use the block format `deprecated::[7.12.0,...]` so it displays as an admon.

Comment (jrodewig) [reply docs/reference/query-dsl/geo-polygon-query.asciidoc:7]: Forgot to mention: Thanks for spotting and fixing this! ❤️

Comment (lockewritesdocs) [reply docs/reference/query-dsl/geo-polygon-query.asciidoc:7]: Thanks @jrodewig! I didn't know that was possible, so appreciate the heads up. I'll make that change.
